### ISSUE TITLE
Loss not decreasing, GTX 1070, cuDNN 5.0 for RC 8.0, Cuda 8.0 RC

### ISSUE THREAD

**Issue by TylerBalsam** [tensorflow/tensorflow#3582]
I've spent about 12 hours monkeying around before throwing in the towel here. I've checked the other similar threads on this and done the steps there. This includes #3068 and #3507. 

### Environment info
Operating System: Ubuntu 14.04

Installed version of CUDA and cuDNN: 
Cuda 8.0 RC
cuDNN 5.0 RC
Output of libcudn*:
/usr/local/cuda/lib64/libcudadevrt.a       /usr/local/cuda/lib64/libcudnn.so
/usr/local/cuda/lib64/libcudart.so         /usr/local/cuda/lib64/libcudnn.so.5
/usr/local/cuda/lib64/libcudart.so.8.0     /usr/local/cuda/lib64/libcudnn.so.5.0.5
/usr/local/cuda/lib64/libcudart.so.8.0.27  /usr/local/cuda/lib64/libcudnn_static.a
/usr/local/cuda/lib64/libcudart_static.a

Built from source
1. The commit hash: 5161e4c51b994b3feb93cdb851479c29a3450f31
2. The output of `bazel version`:
Build label: 0.3.0
Build target: bazel-out/local-fastbuild/bin/src/main/java/com/google/devtools/build/lib/bazel/BazelServer_deploy.jar
Build time: Fri Jun 10 11:38:23 2016 (1465558703)
Build timestamp: 1465558703
Build timestamp as int: 1465558703

### Notes

I've built and thrown out, built and thrown out, cleaned the slate and reinstalled in about as many permutations as I can think. However, with training, the loss freezes. I simplified my network all the way down to a shallow autoencoder where there is only one image in the training set, and still the loss freezes. This worked completely fine using the CPU for similar code from a regular old pip distro. I'm using the MSE loss on a 224*224 image scaled between 0 and 1 to try to keep things as regular as possible.

Something odd to note is that no matter what my loss starts at, it always tends to 11893.53125. Ironically, if it starts below there, it goes up to it. Example of several 40 epoch runs: 11893.52930, 11893.53223, 11893.52930, 11893.52930, 11893.52930.

My theory was that it was outputting 0s -- assuming an even distribution, the summed MSE loss would be 224x224x.5^2 = 12544, which is pretty close to our 11893.52930. Since regular images won't have an even pixel distribution, this makes sense. In practice, after testing the network, it outputs all 1s.

A side note is that when running the trainer test, the lambda value was not always exactly 2 -- every few iterations it was off by a fraction.

I'm really scratching my head here -- any folks with combat experience see any telltale signs of a solution? Thank you very much in advance for your time.

**Comment by jiapei100** [comment]
I've got exactly the same symptom... 
Keras with Tensorflow backend... 
I'll try TF 0.10.0 tonight...

**Comment by TylerBalsam** [comment]
Alright, sounds good. Let me know what you find. It's been almost a week for me.

From what I've seen, I suspect the interaction between CUDA 8.0 and Tensorflow is our primary culprit. Which is sad because that's required for the 10 series cards.

**Comment by kestrelm** [comment]
Hello,

Is anybody going to verify that the new upcoming update is going to run with the 10 series cards? In particular the gtx 1060 and 1070? This is important as I am thinking of getting a gtx 1060 for cntk and the current status of it not running on the 10 series cards makes me pause on switching over to cntk.

**Comment by girving** [comment]
@TylerBalsam Just arriving at this thread now; can you explain why you think this isn't just a model convergence issue?  Nothing in the thread implies there is a bug in TensorFlow to my eyes.

**Comment by aselle** [comment]
Closing automatically due to lack of recent response. Glancing at the thread history, this looks like it is more of a stack overflow question because it is concerning usage of TensorFlow rather than any type of bug. The stackoverflow community is generally experienced and helpful for these types of questions. Good luck!

**Comment by aziabari** [comment]
Is this issue resolved?

I had my CNN network in tensorflow which was working fine when I used either CPU or GPU.
After I updated my Cuda to Cuda 8.0.61, my code doesn't function properly with GPU and the loss value freezes after 2-3 epochs!! (doesn't decrease).
When I use CPU, my code works fine and I don't have any problem and everything is fine!

Please let me know if there is any solution to this issue! 
Thanks
